src/content/en/android/custom-tabs/implementation-guide/index.md

```diff
@@ -48,107 +48,105 @@ customTabsIntent.launchUrl(this, Uri.parse(url));
+Many Android applications use custom View Entrance and Exit animations when transitioning between
+Activities on Android. Custom Tabs is no different, you can change the entrance and exit
 (when the user presses Back) animations to keep them consistent with the rest of your application.
 
 ```java
 builder.setStartAnimations(this, R.anim.slide_in_right, R.anim.slide_out_left);
 builder.setExitAnimations(this, R.anim.slide_in_left, R.anim.slide_out_right);
 ```
 
-## Warm up Chrome to make pages load faster
+## Warm up the browser to make pages load faster
 
-By default, when [`CustomTabsIntent#launchUrl`][7] is called, it will spin up Chrome and launch the
-URL. This can take up precious time and impact on the perception of smoothness.
+By default, when [`CustomTabsIntent#launchUrl`][7] is called, it will spin up the browser and launch
+the URL. This can take up precious time and impact on the perception of smoothness.
 
-We believe that users demand a near instantaneous experience, so we have provided a Service in
-Chrome that your app can connect to and tell Chrome to warm up the browser and the native
-components.  We are also experimenting with the ability for you, the developer to tell Chrome the
-likely set of web pages the user will visit.  Chrome will then be able to perform:
+We believe that users demand a near instantaneous experience, so we have provided a Service that 
+your app can connect to and tell the browser and its native components to warm up.
+Custom Tabs also provide the ability for you, the developer to tell the browser the
+likely set of web pages the user will visit. Browsers will then be able to perform:
 
 - DNS pre-resolution of the main domain
 - DNS pre-resolution of the most likely sub-resources
 - Pre-connection to the destination including HTTPS/TLS negotiation.
 
-The process for warming up Chrome is as follows:
+The process for warming up the browser is as follows:
 
 - Use [`CustomTabsClient#bindCustomTabsService`][8] to connect to the service.
-- Once the service is connected, call [`CustomTabsClient#warmup`][9] to start Chrome behind the
+- Once the service is connected, call [`CustomTabsClient#warmup`][9] to start the browser behind the
 scenes.
 - Call [`CustomTabsClient#newSession`][10] to create a new session. This session is used for all
 requests to the API.
 - Optionally, attach a [`CustomTabsCallback`][11] as a parameter when creating a new session, so
 that you know a page was loaded.
-- Tell Chrome which pages the user is likely to load with [`CustomTabsSession#mayLaunchUrl`][12]
+- Tell the browser which pages the user is likely to load with
+[`CustomTabsSession#mayLaunchUrl`][12]
 - Call the [`CustomTabsIntent.Builder`][4] constructor passing the created
 [`CustomTabsSession`][13] as a parameter.
 
-## Connect to the Chrome Service
+## Connect to the Custom Tabs Service
 
 The [`CustomTabsClient#bindCustomTabsService`][8] method takes away the complexity of connecting to
 the Custom Tabs service.
```

Comment:
The code example just below this section has Chrome specific comments.
```
// Package name for the Chrome channel the client wants to connect to. This
// depends on the channel name.
// Stable = com.android.chrome
// Beta = com.chrome.beta
// Dev = com.chrome.dev
public static final String CUSTOM_TAB_PACKAGE_NAME = "com.android.chrome";  // Change when in stable
```

Should the phrasing be updated for these? e.g.

```
// Taking Chrome channels as an example, a client may specify a 
// browser by package name.
// Stable = com.android.chrome
// Beta = com.chrome.beta
// Dev = com.chrome.dev
public static final String CUSTOM_TAB_PACKAGE_NAME = "com.android.chrome";  // Change when in stable
```

It is also not clear what "Change when in stable" means for these docs.

Reply:
It's on my TO-DO list to update this too, but leaving to a following PR.